src/ch.usi.inf.nodeprof/src/ch/usi/inf/nodeprof/jalangi/factory/AbstractFactory.java

```diff
@@ -79,13 +81,12 @@ protected static String readString(DynamicObject cb, String name) {
         }
     }
 
-    @TruffleBoundary
```

Comment:
The change seems only swapping the two methods' location, let's keep it as it was.